Patterns/Configuration.md

```diff
@@ -77,7 +77,7 @@ When locally developing azure functions you are required to create a `local.sett
 }
 ```
 
-This file helps local developers by setting all values in the `Values` dictionary as environment variables so it is not necessary to add them to your shell. So this list is also a flat dictionary and requires the `:` seperator when adding sections in the key.
+This file helps local developers by setting all values in the `Values` dictionary as environment variables so it is not necessary to add them to your shell. So this list is also a flat dictionary and requires the `:` separator when adding sections in the key.

```

Comment:
We use `__` in the local.settings.json in the values dictionary and that also works.